molgenis-data-annotators/src/test/java/org/molgenis/data/annotation/entity/impl/SnpEffAnnotatorTest.java

```diff
@@ -129,5 +292,25 @@ public DataService dataService()
 			return mock(DataService.class);
 		}
 
+		@Bean
+		public JarRunner jarRunner()
+		{
+			JarRunner jarRunner = mock(JarRunner.class);
+			try
+			{
+				List<String> params = Arrays.asList("-Xmx2g", null, "hg19", "-noStats", "-noLog", "-lof", "-canon",
+						"-ud", "0", "-spliceSiteSize", "5");
+				when(jarRunner.runJar(SnpEffAnnotator.NAME, params, ResourceUtils.getFile("test-edgecases.vcf")))
+						.thenReturn(ResourceUtils.getFile("snpEffOutputCount.vcf"));
+				when(jarRunner.runJar(SnpEffAnnotator.NAME, params, ResourceUtils.getFile("test-snpeff.vcf")))
+						.thenReturn(ResourceUtils.getFile("snpEffOutput.vcf"));
+
+			}
+			catch (Exception e)
+			{
+				e.printStackTrace();
+			}
+			return jarRunner;
+		}
```

Comment:
What I mean, a normal untrained mock.
Can't you reset and train it in the beforeMethod or in the test itself?